solutions/fully-configurable/README.md

```diff
@@ -0,0 +1,109 @@
+# Cloud automation for Event Notifications (Fully configurable)
```

Comment:
to be in consistent with other DAs, let's follow the structure by defining `Prerequisites` as well . Same for security-enforced DA

https://github.com/terraform-ibm-modules/terraform-ibm-scc/blob/main/solutions/fully-configurable/README.md#prerequisites